### ISSUE TITLE
Make tree keyboard navigable

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#608]
`aria-activedescendant` should be one element and not a concatenated string of selected id 

This is used when a composite widget is responsible for managing its current active child to reduce the overhead of having all children be focusable. Examples include: multi-level lists, trees, and grids. In some implementations the user agent may use aria-activedescendant to tell assistive technologies that the active descendant has focus. Authors MAY use the aria-activedescendant attribute on the focused descendant of a composite widget; for example, on a textbox descendant of a combo box.
